lib/sources/monitor/availability_checker.rb

```diff
@@ -1,5 +1,8 @@
 require "sources/monitor/core/api_client"
 require "sources/monitor/core/logging"
+require "active_support/core_ext/object/try"
```

Comment:
why is it needed by this PR? (and below)

Reply:
those were loaded by manageiq-loggers gem and I found that  minimally we need 
`require "active_support/core_ext/string"` because `.blank?` is from this library.
I added also other to have as it was. (we don't break anything)